dimod/generators/anti_crossing.py

```diff
@@ -12,89 +12,102 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-from dimod import BinaryQuadraticModel
+from dimod.binary.binary_quadratic_model import BinaryQuadraticModel
+from dimod.vartypes import Vartype
 
 __all__ = ['anti_crossing_clique', 'anti_crossing_loops']
 
 
-def anti_crossing_clique(num_variables):
-    """Anti crossing problems with a single clique.
+def anti_crossing_clique(num_variables: int) -> BinaryQuadraticModel:
+    """Generate an anti crossing problem with a single clique.
```

Comment:
I find this description misleading, it creates an impression that the entire problem is a clique. Perhaps something like, "Generate an anti-crossing problem of a clique with attached nodes." The single liners doesn't fully describe the problem, of course, but it doesn't misdescribe it.

Reply:
The problem is really with the function name IMO, but we are locked in. Blame @conta877 :laughing:

Reply:
"from cliques"